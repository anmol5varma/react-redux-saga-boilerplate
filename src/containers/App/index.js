import { connect } from 'react-redux';
import { sampleAction } from '../../redux/actions/index.actions';
import App from '../../components/App'

const mapStateToProps = ({ sample }) => ({
    sample,
});

const mapDispatchToProps = dispatch => ({
    updateSampleReducer: (value) => dispatch(sampleAction(value)),
});

export default connect(
    mapStateToProps,
    mapDispatchToProps
)(App);
