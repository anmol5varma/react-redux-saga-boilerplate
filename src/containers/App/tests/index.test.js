import React from 'react';
import { shallow } from 'enzyme';
import App from '../index';

describe('<App />', () => {
  it('renders correctly', () => {
    const actual = shallow(<App store={{}} />);
    expect(actual).toMatchSnapshot();
  });
});
