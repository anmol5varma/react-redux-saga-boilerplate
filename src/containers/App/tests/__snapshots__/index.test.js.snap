// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<Provider
  store={Object {}}
>
  <BrowserRouter>
    <Route
      component={
        Object {
          "mapDispatchToProps": [Function],
          "mapStateToProps": [Function],
          "mockDispatch": [MockFunction],
          "reactComponent": [Function],
        }
      }
      path="/"
    />
  </BrowserRouter>
</Provider>
`;
